pkgs/servers/dns/bind/default.nix

```diff
@@ -13,6 +14,8 @@ stdenv.mkDerivation rec {
     sha256 = "10iwkghl5g50b7wc17bsb9wa0dh2gd57bjlk6ynixhywz6dhx1r9";
   };
 
+  preConfigure = "export AR=${stdenv.cc}/bin/${stdenv.cc.targetPrefix}ar";
```

Comment:
What's this about? Is the setup hook not working?

Reply:
https://gitlab.isc.org/isc-projects/bind9/blob/master/configure.in#L119

Reply:
https://github.com/NixOS/nixpkgs/blob/master/pkgs/build-support/bintools-wrapper/setup-hook.sh#L75

Reply:
Permalink for posterity: https://github.com/NixOS/nixpkgs/blob/806edaa0a20db3358836d55d203500b87dbe8624/pkgs/build-support/bintools-wrapper/setup-hook.sh#L74-L84
(Sorry for the necro)